source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/test/integ.no-arguments.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ *  Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+// Imports
+import { CfnJob } from '@aws-cdk/aws-glue';
+import { Bucket, CfnBucket } from '@aws-cdk/aws-s3';
+import { App, Duration, Stack } from '@aws-cdk/core';
+import { KinesisStreamGlueJob, KinesisStreamGlueJobProps } from '../lib';
+
```

Comment:
Generally the `***.no-arguments.ts` test case is very similar to the minimal deploy able pattern from README.md, but this one looks a lot different

Reply:
Updating the no-arg-integ test.